src/base/traverseFunctions.js

```diff
@@ -1,4 +1,5 @@
 import { LOADED } from './constants.js';
+import { convertCompilerOptionsFromJson } from 'typescript';
```

Comment:
Can you explain what this is for? This project isn't using typescript at the moment.

Reply:
I am sorry about this. Tbh, I have no idea what this is. Never added it myself. It's probably something that VS Code added?